maven-failsafe-plugin/src/main/java/org/apache/maven/plugin/failsafe/IntegrationTestMojo.java

```diff
@@ -130,12 +130,24 @@
     private boolean useFile;
 
     /**
-     * Set this to "true" to cause a failure if none of the tests specified in -Dtest=... are run. Defaults to
+     * Set this to "true" to cause a failure if none of the tests specified in -Dit.test=... are run. Defaults to
      * "true".
      *
      * @since 2.12
+     * @deprecated Since 3.0.0-M8, use "failsafe.failIfNoSpecifiedTests" instead.
      */
+    @Deprecated
     @Parameter( property = "it.failIfNoSpecifiedTests", defaultValue = "true" )
+    private boolean failIfNoSpecifiedTestsDeprecated;
+
+    /**
+     * Set this to "true" to cause a failure if none of the tests specified in -Dit.test=... are run. Defaults to
+     * "true".
+     * Replacing "it.failIfNoSpecifiedTests" to be consistent with surefire plugin.
```

Comment:
as `true` is the default value, I would start documentation from something like

> Set this to false to prevent a failure if none ....

Reply:
I agree with your suggestion, but @liry did not change the wording here.

Since there's no response from @liry, can the PR be merged anyway? In my fork I've rebased the branch to master. Builds were successful: https://github.com/andpab/maven-surefire/actions/runs/4198008135

After the merge of this one I could make a new PR that adjusts the wording according to your suggestion.

Reply:
Sorry, I somehow missed the first comment. Wording changed.